### PR TITLE
Added code for MO's Algorithm in Cpp

### PR DESCRIPTION
**Pull request by sirjan13**
Resolve issue #219

### CODE REVIEW

File: code/Square Root Decomposition/MO's Algorithm/MOAlgo.cpp [hunk 1]
```diff
@@ -0,0 +1,88 @@
+//Problem : http://www.spoj.com/problems/DQUERY/
```
Comment (AdiChat) [inline code/Square Root Decomposition/MO's Algorithm/MOAlgo.cpp]: Kindly remove this link as it is not consistent with our offline mission 👍

Comment (sirjan13) [reply code/Square Root Decomposition/MO's Algorithm/MOAlgo.cpp]: updated PR

**Comment by sirjan13** [comment]
Updated my Pull Request .Removed the link :)

**Comment by AdiChat** [comment]
Looks good 👍 

Hope you are enjoying your journey with **OpenGenus**

**Pull request merged**